src/DataFixtures/SubscriberFixtures.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Subscriber;
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use Faker;
+
+class SubscriberFixtures extends Fixture
+{
+    private const GENDER = [
+        'F',
+        'H',
+    ];
+
+    public function load(ObjectManager $manager)
+    {
+        $faker = Faker\Factory::create('fr_FR');
+        for ($i = 0; $i < 100; $i++) {
+            $subscriber = new Subscriber();
+            $subscriber->setFirstname($faker->firstNameMale);
+            $subscriber->setLastname($faker->lastName);
+            $subscriber->setBirthdate(\DateTime::createFromFormat('Y-m-d', "1980-09-09"));
```

Comment:
tu peux faire $faker->dateTimeThisCentury->format('Y-m-d') 
ainsi tu n'auras pas tjs les memes dates

Reply:
j'ai justement essayé avec faker mais j'ai cette erreur

Argument 1 passed to App\Entity\Subscriber::setBirthdate() must implement interface DateTimeInterface, string given, called in /home/wilder/Bureau/orleans-php-202009-project-aviro  
  n/src/DataFixtures/SubscriberFixtures.php on line 25